test/File/ExcludeMimeTypeTest.php

```diff
@@ -44,13 +48,18 @@ public function basicBehaviorDataProvider()
      * Ensures that the validator follows expected behavior
      *
      * @dataProvider basicBehaviorDataProvider
-     * @return void
+     *
+     * @param string|array $options
+     * @param array $isValidParam
+     * @param bool $expected
+     * @param array $messages
      */
-    public function testBasic($options, $isValidParam, $expected)
+    public function testBasic($options, array $isValidParam, $expected, array $messages = [])
```

Comment:
It's better to have all possible values in the data provider for get the whole picture. Please remove the default value.